db/seeds/initial_seed.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+cohort2021 = Cohort.find_or_create_by!(start_year: 2021)
+
+ambition_cip = CoreInductionProgramme.find_or_create_by!(name: "Ambition Institute")
+edt_cip = CoreInductionProgramme.find_or_create_by!(name: "Education Development Trust")
+teach_first_cip = CoreInductionProgramme.find_or_create_by!(name: "Teach First")
+ucl_cip = CoreInductionProgramme.find_or_create_by!(name: "UCL Institute of Education")
+
+ambition = LeadProvider.find_or_create_by!(name: "Ambition Institute")
+ambition.update!(cohorts: [cohort2021]) unless ambition.cohorts.any?
+LeadProviderCip.find_or_create_by!(lead_provider: ambition, cohort: cohort2021, core_induction_programme: ambition_cip)
```

Comment:
Hmmmm, now that I see it, I wonder if `cohorts` field on `LeadProvider` is duplicating data included in `lead_provider_cips`. And if there is a clever way of avoiding it.

Thoughts?

Reply:
I think so, but I really don't want to change that here. That seems complex

Reply:
Ech, it does indeed. Can we have a tech-debt ticket for it?

Reply:
https://trello.com/c/IKKuN8dC/384-tech-debt-consider-merging-cohortsleadproviders-and-leadprovidercips-to-eliminate-duplication-of-data